packages/interbit/src/chainManagement/configureChains.js

```diff
@@ -7,13 +7,21 @@ const {
   }
 } = require('interbit-covenant-tools')
 
+const log = require('../log')
 const configureJoins = require('./configureJoins')
 
 process.on('unhandledRejection', reason => {
-  console.log(`Caught unhandled rejection. Reason: ${reason}`)
-  console.log(reason)
+  log.error(`Caught unhandled rejection. Reason: ${reason}`)
+  log.error(reason)
 })
 
+/**
+ * Configures the chains on a node to the specifications in the Interbit
+ * manifest file passed as `interbitManifest`. Applies covenants, configures joins, and
```

Comment:
This line is longer than 80 columns. Do you think this needs fixing?

I'm okay with long lines in source comments, but I'll have to re-wrap them for the transformed documentation markup because there's doc CI that checks for lines longer than 80 columns.

Reply:
We don't actually have a linter rule for comment length therefore I don't think it needs fixing. If it does become a problem in the future as we document more things we can always revisit the linter rules.